src/influx_udp_sup.erl

```diff
@@ -18,22 +18,21 @@
 start_link() ->
   supervisor:start_link({local, ?MODULE}, ?MODULE, []).
 
--spec start_pool(Host::inet:ip_address()|inet:ip_hostname(), Port::inet:port_number()) -> ok.
-
-start_pool(Host, Port) ->
+-spec start_pool(
+    Host::inet:ip_address()|inet:ip_hostname(), Port::inet:port_number()
+  ) -> {ok, undefined | pid()} | {ok, undefined | pid(), any()} | {error, any()}.
```

Comment:
Line 23 is too long:   ) -> {ok, undefined | pid()} | {ok, undefined | pid(), any()} | {error, any()}..